openedx/openedx-platform
# incorporated manual verification

Work description:

1: Model class ManualVerification(IDVerificationAttempt): with reason field
2: Modified IDVerificationService such that it uses the new class 
3: Added management command to add the provided users to the ManualVerification model

Test Instructions: 

1: Create a new user
2: Run the manual verification management command
usage: manage.py manual_verifications [-h] [--version] [-v {0,1,2,3}]
                                      [--settings SETTINGS]
                                      [--pythonpath PYTHONPATH] [--traceback]
                                      [--no-color] --email-ids-file
                                      EMAIL_IDS_FILE
 e.g.
./manage.py lms manual_verifications --email-ids-file /edx/app/edxapp/edx-platform/email.txt --settings=aws
3: Verify that the new users are verified i.e. they do not see a verify button
